client/ayon_core/host/dirmap.py

```diff
@@ -190,10 +190,17 @@ def _get_local_sync_dirmap(self):
             if remote_provider != "local_drive":
                 remote_site = "studio"
             for root_name, active_site_dir in active_roots_overrides.items():
-                remote_site_dir = (
-                    remote_roots_overrides.get(root_name)
-                    or sync_settings["sites"][remote_site]["root"][root_name]
-                )
+                # If my remote is SFTP or other kind of remote use work dir as path
+                if remote_provider == "sftp":
```

Comment:
Shouldn't we handle only those overrides that actually can affect this? That is question for @kalisp .